From f29a7fb5907c09c53784886997899d5f36c37e86 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?P=C3=A1draig=20Brady?= <P@draigBrady.com>
Date: Thu, 1 Sep 2016 16:39:04 +0100
Subject: [PATCH] doc: fix typo in date example

* doc/coreutils.texi (date invocation): Add a missing quotation mark.
Fixes http://bugs.gnu.org/24349

Upstream-commit: 22d188e2991ce64445ba89dbb32e3c6bec42e235
Signed-off-by: Kamil Dudka <kdudka@redhat.com>
---
 doc/coreutils.texi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/doc/coreutils.texi b/doc/coreutils.texi
index a58339a..88d6506 100644
--- a/doc/coreutils.texi
+++ b/doc/coreutils.texi
@@ -15467,7 +15467,7 @@ of the month, you can use the (GNU extension)
 the padding altogether:
 
 @example
-date -d 1may '+%B %-d
+date -d 1may '+%B %-d'
 @end example
 
 @item
-- 
2.17.2

